index.js

```diff
@@ -21,7 +21,7 @@ module.exports = function() {
 
           if (existsSync(fileAbsPath)) {
             const rawMd = readFileSync(fileAbsPath, 'utf-8');
-            node.children = unified.parse(rawMd);
+            node.children = unified.parse(rawMd).children;
```

Comment:
I made this change as otherwise the content appeared empty in the test. I confirm it still works with this change in the docusaurus site where I am using this module; you may wish to confirm the same.